### PR TITLE
Updating download links for VS2022

### PR DESCRIPTION
**Pull request by andysterland**
Adding links to directly download Visual Studio with the right workloads for the asp.net tutorials.

### CODE REVIEW

**Comment by dnfadmin** [comment]
[CLA assistant check](https://cla.dotnetfoundation.org/dotnet/AspNetCore.Docs?pullRequest=25391) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/dotnet/AspNetCore.Docs?pullRequest=25391) before we can accept your contribution.<br/><br/>:x: andysterland [sign now](https://cla.dotnetfoundation.org/dotnet/AspNetCore.Docs?pullRequest=25391)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/dotnet/AspNetCore.Docs?pullRequest=25391) it.</sub>

File: aspnetcore/tutorials/first-mvc-app/start-mvc.md [hunk 1]
```diff
@@ -32,7 +32,8 @@ At the end of the series, you'll have an app that manages and displays movie dat
 
 # [Visual Studio](#tab/visual-studio)
 
-[!INCLUDE[](~/includes/net-prereqs-vs-6.0.md)]
+* [Download Visual Studio 2022 Community with the ASP.NET and web development workload](https://visualstudio.microsoft.com/thank-you-downloading-visual-studio/?version=VS2022&sku=Community&channel=Release&source=VSFeaturesPage&add=Microsoft.VisualStudio.Workload.ManagedDesktop&add=Microsoft.VisualStudio.Workload.NetWeb&add=Microsoft.VisualStudio.Workload.Azure&focusedui=true&cid=4005) - Free for students and open source developers.
+* [Download Visual Studio 2022 Professional with the ASP.NET and web development workload](https://visualstudio.microsoft.com/thank-you-downloading-visual-studio/?version=VS2022&sku=Community&channel=Release&source=VSFeaturesPage&add=Microsoft.VisualStudio.Workload.ManagedDesktop&add=Microsoft.VisualStudio.Workload.NetWeb&add=Microsoft.VisualStudio.Workload.Azure&focusedui=true&cid=4005) - Free 30 day trial for professional developers.
```
Comment (Rick-Anderson) [inline aspnetcore/tutorials/first-mvc-app/start-mvc.md:36]:
```suggestion
[!INCLUDE[](~/includes/net-prereqs-vs-6.0.md)]
```

Comment (Rick-Anderson) [reply aspnetcore/tutorials/first-mvc-app/start-mvc.md:36]: @andysterland updates need to be made to the include file because it's used in many tutorials.

**Review by guardrex — COMMENTED** [review]
A couple of concerns with this ...

* The landing page at https://visualstudio.microsoft.com/vs/#download provides a DDL for selecting the VS SKU. I defer to the team, but I don't see the need to split out the SKUs like this. Besides, what about Enterprise? One would need to list that as well.
* There is a strong prohibition about tagging anything MS releases as "free" or for any particular terms, such as a 30-day trial. My understanding from my .NET docs days is that they don't want documentation to commit the marketing/product units to anything specific about the terms of use for any particular SKU. I think they'd object to this being in the topic.

**Review by wadepickett — COMMENTED** [review]
I suggest limiting the change here to just the workload instruction, as opposed to a collection of more specific links. Use the original link but change it a bit to something like:  "**Visual Studio 2022**. Select the **ASP.NET and web development workload** during installation.

I agree the previous instruction would result in some confusion and extra clicks once folks arrive at the VS 2022 marketing page and needs help, and I am glad this was brought up.  I suspect however, that it would be better to leave the VS 2022 link as is, but instead, clarify where the workload selection occurs.   As it was originally, it could be interpreted to mean that they should look for a download link on the marketing page that specifies VS with the ASP.NET and web development workload. 

With that change I feel the path is fairly quick and clear and addresses the important concerns @guardrex pointed out.

**Comment by Rick-Anderson** [comment]
@andysterland  did you have a reason for your change? We've probably had over 10 million folks follow those instructions and I can't remember any of them having a problem. We can reopen this if you have a compelling reason. Not sure why  you're not approved

image

**Pull request closed without merging**